docs/developers.rst

```diff
@@ -11,7 +11,7 @@ Overview of Source Distribution
 
  * ``docs/`` -- Source code for documentation.
  * ``libxmp/`` -- Source files for XMP Toolkit
- * ``setup.py`` -- Distutils configuration file.
+ * ``pyproject.toml`` -- Flit configuration file.
```

Comment:
This is not specific to flit, so there's no need to specify that flit is used. This is a packaging cofniguration file.

Reply:
OK